centrifuge-app/src/components/EpochList.tsx

```diff
@@ -35,19 +36,21 @@ const columns: Column[] = [
 export const EpochList: React.FC<Props> = ({ pool }) => {
   const { data: metadata } = usePoolMetadata(pool)
 
-  const investments: TableDataRow[] = pool.tranches.map((token) => {
+  const investments: TableDataRow[] = pool.tranches.map((token, index) => {
     const trancheMeta = metadata?.tranches?.[token.id]
+    const price = token.tokenPrice?.toDecimal() || Dec(0)
     return {
       order: `${trancheMeta?.symbol} investments`,
-      locked: token.outstandingInvestOrders?.toDecimal() || new Decimal(0),
+      locked: token.outstandingInvestOrders?.toDecimal().mul(price) || new Decimal(0),
```

Comment:
I think it would be good to add a `toTokenBalance(price)` method to `CurrencyBalance`, so this can be changed to
```suggestion
      locked: token.outstandingInvestOrders?.toTokenBalance(price).toDecimal() || new Decimal(0),
```

Reply:
turns out this is also already handled in cent-js (got lost in merge conflicts), I'll keep an eye out in case I see something like this again and will implement the method `toTokenBalance()` then 👍